src/options/base_options.toml

```diff
@@ -63,16 +63,18 @@ name   = "Base"
   category   = "common"
   short      = "v"
   long       = "verbose"
-  type       = "void"
-  handler    = "increaseVerbosity"
+  type       = "bool"
```

Comment:
If we declare those options to be of `type` `bool`, don't we also start accepting `--no-verbose`?

Reply:
note the `alternate = false` below. This prevents exactly that.

Reply:
Ah, great!